sdk/src/beta9/config.py

```diff
@@ -176,6 +180,7 @@ def prompt_for_config_context(
     prompt_gateway_port = functools.partial(
         terminal.prompt, text="Gateway Port", default=gateway_port or settings.gateway_port
     )
+    prompt_tls = functools.partial(terminal.prompt, text="TLS", default=False)
```

Comment:
why is this prompted? shouldn't it be using that function below?

is_tls_enabled

Reply:
removed